app/models/setting.rb

```diff
@@ -270,8 +273,8 @@ def self.load_defaults
     true
   end
 
-  def self.set(name, description, default, full_name = nil, value = nil)
-    {:name => name, :value => value, :description => description, :default => default, :full_name => full_name}
+  def self.set(name, description, default, full_name = nil, value = nil, encrypted = nil)
```

Comment:
Now is a good time to add an options hash containing encrypted to the end of this method signature, it's getting overly long and specific - most settings will not be encrypted.